WordPress/src/test/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterViewModelTest.kt

```diff
@@ -1,51 +1,131 @@
 package org.wordpress.android.ui.activitylog.list.filter
```

Comment:
Love (❤️ ): Loving the tests, and actually this PR response to my previous PR are testing view model functionality. Many thanks for that and the dummy provider!

PS: Just as an FYI, on some tests the `..., when ...` is missing.

Reply:
Fixed in 4d51eec8084a0acf0c3ffec5e12f0011a84e255f

> Just as an FYI, on some tests the ..., when ... is missing.

I felt like that if the "Arrange" and "Act" sections don't have any "when", it's ok to omit the "when" in the name. However, it's probably better to always keep it ;).

Reply:
Yea, definitely, if your intention is to sometimes avoid the `when`, by all means,  just let me know.  My commenting is purely because I was thinking that it has split your mind by mistake.